engine/src/deploy_sign.cpp

```diff
@@ -1034,7 +1034,8 @@ static bool MCDeploySignWindowsAddTimeStamp(const MCDeploySignParameters& p_para
 	// the sig.
 	if (t_request != nil)
 	{
-		t_request -> blob -> signature = nil;
+        if (t_request -> blob != nil)
```

Comment:
Ah - it turns out that I was reading the code incorrectly because of mixed tab/space indenting. Sorry for wasting your time. :disappointed: